### ISSUE TITLE
Does Workmanager is better alternative for android-priority-jobqueue ?

### ISSUE THREAD

**Issue by shakil807g** [yigit/android-priority-jobqueue#458]

**Comment by biliboss** [comment]
@shakil807g yeas. But the last time that i check, Workmanager was in Alpha and wasn't production ready yet. 

Priority Jobqueue appears "abandoned" by @yigit - i don't know if have a new good fork.

**Comment by JimVanG** [comment]
Yigit is focusing all his time on Google and their improvements, he’s an amazing programmer.

@shakil807g,  I’d honestly go with what Android is doing. Which is basically this but native and well supported and I can tell you now, will be the future.

**Comment by yigit** [comment]
@biliboss sorry to disappoint you. I could've been more responsive here thought the work is quite overwhelming that i've been neglecting the project. Also, WorkManager is designed to replace JobManager. They don't do the same work but i do think JobManager does more than it sohuld in today's world (it was a very different environment when we created this library 7 years ago).

I've already updated the readme to deprecate. Happy to fix any important issue here so people who are using the library can continue but other than that, i don't plan to do any improvements here.
